site/components/Navigation/index.tsx

```diff
@@ -58,47 +72,360 @@ export const Navigation: FC<Props> = ({
+              active={activePage === "Disclaimer"}
+              url="/disclaimer"
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "DOCS",
+                id: "shared.docs",
+              })}
+              key="docs"
+              url={urls.officialDocs}
+            />,
+          ]}
         />
       </HeaderDesktop>
 
       <HeaderMobile
-        buttons={[
-          <ChangeLanguageButton key="ChangeLanguageButton" />,
-          ...(showThemeToggle ? [<ThemeToggle key="ThemeToggle" />] : []),
-        ]}
+        buttons={[<ChangeLanguageButton key="ChangeLanguageButton" />]}
       >
-        <NavItemMobile
-          url={createLinkWithLocaleQuery(urls.offset, locale)}
-          name={t({ message: "Offset", id: "shared.offset" })}
-        />
-        <NavItemMobile
-          url="/infinity"
-          name={t({ message: "Infinity", id: "shared.infinity" })}
-        />
-        <NavItemMobile
-          url={urls.loveletter}
-          name={t({ message: "Love Letters", id: "shared.loveletters" })}
-        />
-        <NavItemMobile
-          url="/blog"
-          name={t({ message: "Resources", id: "shared.resources" })}
-        />
+        <div className={styles.navMain_MobileItemsWrapper}>
+          <div className="links">
+            <NavItemMobile
+              name={t({ message: "About", id: "shared.about" })}
+              toggledNavItemID={toggledNavItemID}
+              setToggledNavItemID={setToggledNavItemID}
+              id="About"
+              subMenu={[
+                <NavItemMobile
+                  name={t({
+                    message: "BLOG",
+                    id: "shared.blog",
+                  })}
+                  key="blog"
+                  url="/blog"
+                  active={activePage === "Blog"}
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "COMMUNITY",
+                    id: "shared.community",
+                  })}
+                  key="community"
+                  active={activePage === "Community"}
+                  url="/community"
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "CONTACT US",
+                    id: "shared.contact",
+                  })}
+                  key="contact"
+                  active={activePage === "Contact"}
+                  url="/contact"
+                />,
+              ]}
+            />
+            <NavItemMobile
+              name={t({ message: "App", id: "shared.app" })}
+              toggledNavItemID={toggledNavItemID}
+              setToggledNavItemID={setToggledNavItemID}
+              id="App"
+              subMenu={[
+                <NavItemMobile
+                  name={t({ message: "BUY KLIMA", id: "shared.buy" })}
+                  key="buy klima"
+                  url={createLinkWithLocaleQuery(urls.buy, locale)}
```

Comment:
Nextjs should do this by default when we use `Link`, so we only need this util when passing between the `app` and the `site`
```suggestion
                  url="/buy"
```

edit: looks like all the other ones are OK

Reply:
fixed